scopes/react/react/templates/react-provider/files/docs.ts

```diff
@@ -0,0 +1,65 @@
+import { ComponentContext } from '@teambit/generator';
+
+export const docsFile = (context: ComponentContext) => {
+  const { name } = context;
+
+  return {
+    relativePath: `${name}.docs.mdx`,
+    content: `---
+description: A customised React component development environment with configurable providers
+labels: ['env', 'extension', 'example', 'providers', 'context']
+---
+
+## Overview
+
+A customised React development environment for building and rendering modular and reusable MFEs. Using this customised environment you can override 
+environment configs such as tsconfig or even build tasks, as well as add wrapping contexts for the compositions of components which use this environment.
+
+### Environment config
+To configure the config variables (which can be used, for instance, to configure your providers including api contexts and more) add your configuration to the value object
+of the environment when applying it for your components. See the usage instructions below for an example.
+
+### Usage instructions
+
+Create a **variant** in project's \`workspace.json\` file.
+Set this extension as the variant's environment, for instance for the variant "any components in the 'ui' namespace":
+
+\`\`\`json
+{
+   "teambit.workspace/variants": {
+     "{ui/**}": { // applies this environment for all components with ui namespace and sub-namespaces
+       "org-name.scope-name/namespace/s/${name}": {
+         "configVar1": "value1",  // values here will override the environment's default config values
```

Comment:
`// values here will override the environment's default config values`
this is definitely not true. there is no relation at all between the default config and the config here.
the default config are overridden only by aspect config in the workspace level, not in the variant level.

Also, If I'm remember correctly, adding config here has no point, since you don't have access to them. but I need to check it to make sure

Reply:
So what does this config do? I'm calling the aspect with a config (at least that's what it looks like)
If this doesnt work, and there's no actual scenario where we'd call an env aspect at the workspace level, then what is the point of this config (the config setup is from Ran's initial example for providers)

Reply:
Fixed after further discussion with Gilad